apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue

```diff
@@ -171,26 +176,48 @@ const drawChart = (chartData) => {
             baseAxis: yAxis,
             stacked: true,
             fill: _legend.color,
+            stroke: _legend.color,
         };
         const series = chartHelper.createXYColumnSeries(chart, seriesSettings);
         series.columns.template.setAll({
             height: 20,
         });
-        // series.bullets.push(() => {
-        //     const label = chartHelper.createLabel({
-        //         text: `{${_legend.name}}`,
-        //         populateText: true,
-        //         textAlign: 'end',
-        //         centerX: percent(0),
-        //         centerY: percent(50),
-        //         fontSize: 14,
-        //         direction: 'ltr',
-        //     });
-        //     return chartHelper.createBullet({
-        //         sprite: label,
-        //         dynamic: true,
-        //     });
-        // });
+        series.bullets.push(() => {
+            const label = chartHelper.createLabel({
+                text: `{${_legend.name}}`,
+                populateText: true,
+                textAlign: 'end',
+                centerY: percent(50),
+                fontSize: 14,
+            });
+            return chartHelper.createBullet({
+                locationX: 1,
+                locationY: 0.5,
+                sprite: label,
+                dynamic: true,
+            });
+        });
+        series.columns.template.onPrivate(('width'), (width, target) => {
+            array.each(target?.dataItem?.bullets, (bullet) => {
+                if (width && width < 30) {
+                    bullet.get('sprite').hide();
+                }
+            });
+        });
+        const tooltip = chartHelper.createTooltip();
+        tooltip.label.adapters.add('text', (text, target) => {
+            let _text = '';
+            let totalValue = 0;
+            chart.series.each((s) => {
+                const fieldName = s.get('valueYField') || s.get('valueXField') || '';
+                const value = target.dataItem?.dataContext?.[fieldName];
+                totalValue += value;
+                _text += `\n[${s.get('stroke')?.toString()}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.get('name')}:[/] [bold; fontSize: 14px]${value}[/]`;
+            });
+            _text = `Total: [bold; fontSize: 14px]${totalValue}[/]${_text}`;
+            return _text;
```

Comment:
description: refer to [amcharts5 formatters text-styling guide](https://www.amcharts.com/docs/v5/concepts/formatters/text-styling/#Available_style_codes).